src/UIAutomation/UiaOperationAbstraction/UiaTypeAbstractionEnums.g.h

```diff
@@ -1,146 +1,216 @@
-        SayAsInterpretAs,
-        winrt::Microsoft::UI::UIAutomation::AutomationSayAsInterpretAs,
-        winrt::Microsoft::UI::UIAutomation::AutomationRemoteSayAsInterpretAs>;
-    using UiaScrollAmount = UiaEnum<
-        ScrollAmount,
-        winrt::Microsoft::UI::UIAutomation::AutomationScrollAmount,
-        winrt::Microsoft::UI::UIAutomation::AutomationRemoteScrollAmount>;
-    using UiaStyleId = UiaEnum<
-        int,
-        winrt::Microsoft::UI::UIAutomation::AutomationStyleId,
-        winrt::Microsoft::UI::UIAutomation::AutomationRemoteStyleId>;
-    using UiaSupportedTextSelection = UiaEnum<
-        SupportedTextSelection,
-        winrt::Microsoft::UI::UIAutomation::AutomationSupportedTextSelection,
-        winrt::Microsoft::UI::UIAutomation::AutomationRemoteSupportedTextSelection>;
-    using UiaSynchronizedInputType = UiaEnum<
-        SynchronizedInputType,
-        winrt::Microsoft::UI::UIAutomation::AutomationSynchronizedInputType,
-        winrt::Microsoft::UI::UIAutomation::AutomationRemoteSynchronizedInputType>;
-    using UiaTextAttributeId = UiaEnum<
-        TEXTATTRIBUTEID,
-        winrt::Microsoft::UI::UIAutomation::AutomationTextAttributeId,
-        winrt::Microsoft::UI::UIAutomation::AutomationRemoteTextAttributeId>;
-    using UiaTextDecorationLineStyle = UiaEnum<
-        TextDecorationLineStyle,
-        winrt::Microsoft::UI::UIAutomation::AutomationTextDecorationLineStyle,
-        winrt::Microsoft::UI::UIAutomation::AutomationRemoteTextDecorationLineStyle>;
-    using UiaTextPatternRangeEndpoint = UiaEnum<
-        TextPatternRangeEndpoint,
-        winrt::Microsoft::UI::UIAutomation::AutomationTextPatternRangeEndpoint,
-        winrt::Microsoft::UI::UIAutomation::AutomationRemoteTextPatternRangeEndpoint>;
-    using UiaTextUnit = UiaEnum<
-        TextUnit,
-        winrt::Microsoft::UI::UIAutomation::AutomationTextUnit,
-        winrt::Microsoft::UI::UIAutomation::AutomationRemoteTextUnit>;
-    using UiaToggleState = UiaEnum<
-        ToggleState,
-        winrt::Microsoft::UI::UIAutomation::AutomationToggleState,
-        winrt::Microsoft::UI::UIAutomation::AutomationRemoteToggleState>;
-    using UiaWindowInteractionState = UiaEnum<
-        WindowInteractionState,
-        winrt::Microsoft::UI::UIAutomation::AutomationWindowInteractionState,
-        winrt::Microsoft::UI::UIAutomation::AutomationRemoteWindowInteractionState>;
-    using UiaWindowVisualState = UiaEnum<
-        WindowVisualState,
-        winrt::Microsoft::UI::UIAutomation::AutomationWindowVisualState,
-        winrt::Microsoft::UI::UIAutomation::AutomationRemoteWindowVisualState>;
-    using UiaZoomUnit = UiaEnum<
-        ZoomUnit,
-        winrt::Microsoft::UI::UIAutomation::AutomationZoomUnit,
-        winrt::Microsoft::UI::UIAutomation::AutomationRemoteZoomUnit>;
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// WARNING: Please don't edit this file. It was generated.

```

Comment:
Well... this file is changed a lot by me. Do we need to do anything about the code generator or should we remove this line?

Reply:
Let's keep the line for now until we figure out exactly what the fate of the code generator should be.